commit 0776804c3f37250bd0e442197d4cbae820d4d09a
Author: Nicolas Morey-Chaisemartin <nmoreychaisemartin@suse.com>
Date:   Tue Sep 13 15:49:51 2022 +0200

    btl: openib: Add support for newer hardware
    
    - ConnectX4 VF
    - ConnectX5 VF
    - ConnectX6 VF
    - New broadcom PCI Ids
    
    Signed-off-by: Nicolas Morey-Chaisemartin <nmoreychaisemartin@suse.com>

diff --git opal/mca/btl/openib/mca-btl-openib-device-params.ini opal/mca/btl/openib/mca-btl-openib-device-params.ini
index faf1244fb3a3..b9949de235b8 100644
--- opal/mca/btl/openib/mca-btl-openib-device-params.ini
+++ opal/mca/btl/openib/mca-btl-openib-device-params.ini
@@ -174,7 +174,7 @@ max_inline_data = 256
 
 [Mellanox ConnectX4]
 vendor_id = 0x2c9,0x5ad,0x66a,0x8f1,0x1708,0x03ba,0x15b3,0x119f
-vendor_part_id = 4115
+vendor_part_id = 4115,4116,4117,4118
 use_eager_rdma = 1
 mtu = 4096
 max_inline_data = 256
@@ -183,7 +183,16 @@ max_inline_data = 256
 
 [Mellanox ConnectX5]
 vendor_id = 0x2c9,0x5ad,0x66a,0x8f1,0x1708,0x03ba,0x15b3,0x119f
-vendor_part_id = 4119,4121
+vendor_part_id = 4119,4120,4121,4122
+use_eager_rdma = 1
+mtu = 4096
+max_inline_data = 256
+
+############################################################################
+
+[Mellanox ConnectX6]
+vendor_id = 0x2c9,0x5ad,0x66a,0x8f1,0x1708,0x03ba,0x15b3,0x119f
+vendor_part_id = 4123,4124
 use_eager_rdma = 1
 mtu = 4096
 max_inline_data = 256
@@ -336,7 +345,7 @@ max_inline_data = 72
 
 [Broadcom BCM57XXX]
 vendor_id = 0x14e4
-vendor_part_id = 0x1605,0x1606,0x1614,0x16c0,0x16c1,0x16ce,0x16cf,0x16d6,0x16d7,0x16d8,0x16d9,0x16df,0x16e2,0x16e3,0x16e5,0x16eb,0x16ed,0x16ef,0x16f0,0x16f1,0x1750
+vendor_part_id = 0x1605,0x1606,0x1614,0x16c0,0x16c1,0x16ce,0x16cf,0x16d6,0x16d7,0x16d8,0x16d9,0x16df,0x16e2,0x16e3,0x16e5,0x16eb,0x16ed,0x16ef,0x16f0,0x16f1
 use_eager_rdma = 1
 mtu = 1024
 receive_queues = P,65536,256,192,128
